src/main/java/me/contaria/seedqueue/SeedQueue.java

```diff
@@ -299,10 +276,6 @@ public static void start() {
                 throw new IllegalStateException("Tried to start SeedQueue but a queue is already active!");
             }
 
-            if (!shouldStart()) {
-                return;
-            }
-
```

Comment:
I told you to move this but I realized that this would cause the config to be reloaded on every reset if `shouldStart` returns false so it needs a more proper fix, please revert it for now